doc/user/content/manage/user-management/rbac-newrole.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Create a New User and Role"
+description: ""
+menu:
+  main:
+    parent: user-management
+    weight: 16
+---
+
+This guide walks you through creating a new user and new role in Materialize. By
+the end of this tutorial you will:
+
+* Invite a new user in the Materialize Console
+* Create a new role in your Materialize account using `psql`
+* Apply privileges to the new role
+* Assign the new role to the new user
+
+## Before you begin
+
+In this guide, you'll use `psql` to manage users in Materialize, so be sure you
+have `psql` installed locally.
+
+If you have not signed up for a free trial, [sign up now](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+
+## Invite a new user
+
+First, login to your Materialize console and navigate to Account > Account
+Settings > Users.
+
+Click **Invite User** and fill in the user information.
+
+The **Organization Admin** and **Organization Member** roles refer to console
+privileges and are not related to RBAC roles.
+
+## Create a new role
+
+Materialize stores role names, IDs, and object IDs in the `mz_roles` catalog
+table. You need to connect to your Materialize instance with `psql` to add roles
+to this table.
+
+1. In the [Materialize UI](https://console.materialize.com/), enable the region where you want to run Materialize.
+
+1. On the **Connect** screen, create a new app password and then copy the `psql` command.
+
+    The app password will be displayed only once, so be sure to copy the password somewhere safe. If you forget your password, you can create a new one.
+
+1. Open a new terminal window, run the `psql` command, and enter your app password.
+
+    Now that you're logged in, you can create a new role in the `mz_roles`
+    table.
```

Comment:
It's probably worth mentioning that by logging in for the first time, a new role is automatically created with the same name as the frontegg account name.